base/deprecated.jl

```diff
@@ -190,3 +190,25 @@ MPFR.BigFloat(x::Real, prec::Int, rounding::RoundingMode) = BigFloat(x, rounding
 end
 
 # END 1.3 deprecations
+
+# BEGIN 1.5 deprecations
+"""
+    isimmutable(v) -> Bool
+!!! warning
+    Consider using `!ismutable(v)` instead, as `isimmutable(v)` will be replaced by `!ismutable(v)` in a future release.
```

Comment:
Would be good if this mentioned that ismutable was introduced in Julia v1.5.

Reply:
added in (Since Julia 1.5) in a252f5e